contents/docs/web-analytics/dashboard.mdx

```diff
@@ -0,0 +1,128 @@
+
+| Channel Type     | Description                                                                                     |
+|------------------|-------------------------------------------------------------------------------------------------|
+| Direct           | User typed in the URL directly or used a saved link.                                            |
+| Paid Search      | User clicked on an ad from a search engine, e.g. Google, Bing, or Baidu.                        |
+| Paid Social      | User clicked on an ad from a social media platform, e.g. Facebook, LinkedIn, or Twitter         |
+| Paid Video       | User clicked on an ad from a video platform, e.g. YouTube or Twitch.                            |
+| Paid Shopping    | User clicked on an ad from a shopping platform, e.g. Amazon or eBay.                            |
+| Paid Other       | User clicked on an ad from an unknown platform.                                                 |
+| Cross-Network    | User clicked on a cross-network ad                                                              |
+| Organic Search   | User clicked on a non-ad search result from a search engine, e.g. Google, Bing, or Baidu.       |
+| Organic Social   | User clicked on a non-ad link from a social media platform, e.g. Facebook, LinkedIn, or Twitter |
+| Organic Video    | User clicked on a non-ad link from a video platform, e.g. YouTube or TikTok.                    |
+| Organic Shopping | User clicked on a non-ad link from a shopping platform, e.g. Amazon or eBay.                    |
+| Affiliate        | User clicked on an affiliate link.                                                              |
+| Referral         | User clicked on a referral link.                                                                |
+| Email            | User clicked on a link from an email.                                                           |
+| Display          | User clicked on a display ad, e.g. an ad on Google Display Network.                             |
+| SMS              | User clicked on a link from an SMS.                                                             |
+| Audio            | User clicked on an audio ad, e.g. a podcast ad.                                                 |
+| Push             | User clicked on a push notification.                                                            |
+| Other            | User clicked on a link from an unknown source.                                                  |
+
+<ProductScreenshot
+  imageLight = "https://res.cloudinary.com/dmukukwp6/image/upload/v1711580001/posthog.com/contents/images/docs/web-analytics/dashboard/channels-light.png" 
+  imageDark = "https://res.cloudinary.com/dmukukwp6/image/upload/v1711580003/posthog.com/contents/images/docs/web-analytics/dashboard/channels-dark.png"
+  alt="Web analytics channel types" 
+  classes="rounded"
+/>
+
+> **Read more:** [How channel type is calculated](/docs/data/channel-type)
```

Comment:
I added a big table here for how this is calculated, but it was a bit too big for this page so I moved it to its own page